business/workloads.go

```diff
@@ -148,6 +148,33 @@ func (in *WorkloadService) getWorkloadValidations(authpolicies []*security_v1.Au
 	return validations
 }
 
+// GetAllWorkloads fetches all workloads across every namespace in the cluster.
+func (in *WorkloadService) GetAllWorkloads(ctx context.Context, cluster string) (models.Workloads, error) {
```

Comment:
If this method is just used from `istio_status` to check for the gateways, maybe it could be renamed to "GetAllGateways" and return just the workloads which are gateways? Maybe is a small improvement but, in case there are lots of workloads, we could save a loop here.

Reply:
There's at least one other place where we do the same thing as we are here https://github.com/kiali/kiali/blob/818cb89384dd46a240db8070a3e42a4e305bce50/handlers/workloads.go#L93-L116 so I thought this method could be used more widely but that being said I didn't actually update `handlers/workloads`.